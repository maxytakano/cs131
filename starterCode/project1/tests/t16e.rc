function : int* foo()
{
  return nullptr;
}

function : void main()
{
  delete foo();
}
