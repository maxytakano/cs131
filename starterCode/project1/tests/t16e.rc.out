Error, "/tmp/cs131_testrunner_zITaux/file.rc", line 9: 
  Operand to "delete" is not a modifiable L-value.
Compile: failure.
